theano/compile/debugmode.py

```diff
@@ -620,37 +620,9 @@ def _optcheck_fgraph(input_specs, output_specs, accept_inplace=False):
     equivalence_tracker = _VariableEquivalenceTracker()
     fgraph = gof.fg.FunctionGraph(orig_inputs, orig_outputs,
                                   features=[equivalence_tracker])
```

Comment:
I think the way to do it is simpler. Restart from master. Then the full code for this method is:
```
def _optcheck_fgraph(...):
    fgraph, updates = std_fgraph(input_specs, output_specs, accept_inplace)
    fgraph.attach_feature(_VariableEquivalenceTracker())
    return fgraph, updates
```